addons/mass_mailing_sms/data/mailing_demo.xml

```diff
@@ -121,14 +121,17 @@
         <value name="mailing_trace_id" eval="ref('mailing_sms_0_trace_2')"/>
     </function>
 
+    <record id="mailing_sms_0_utm_source" model="utm.source">
```

Comment:
It is necessary ? Won't it be generated with the mixin ?

Reply:
Hum yes, this one is way more tricky than you may think. When you have an inherits, it will create automatically the record and generate an xmlid. But as we use a mixin now, the xmlid is not automatically generated.

If you pay attention, I have `mailing_sms_0_utm_source` and not `mailing_sms_1_utm_source`, it's because I wanted to match the automatically generated xmlid.